package.json

```diff
@@ -17,14 +17,13 @@
     "build-trezor": "webpack --config webpack/trezor.js --progress=profile --color",
     "build-preload": "webpack --config webpack/preload.prod.js --progress=profile --color",
     "build": "yarn build-trezor && yarn build-preload &&  yarn build-main && yarn build-renderer",
-    "dexsite": "node ./scripts/dexsite.js",
```

Comment:
There's no more dex "site" to copy.  The go build embeds the files that are checked into the release branch.